Automattic/newspack-theme
# Remove compiled CSS files

### All Submissions:






### Changes proposed in this Pull Request:

This PR removes the compiled CSS files from GitHub, and sets the GitHub repo to ignore them. That way, when we create PRs, there won't be all the noise from the multiple CSS files that can be updated (styles, RTL, editor, style pack...) -- instead, you just see the Sass changes, which are much easier to parse. 

To fully test going forward, we'll need to generate the CSS in our test environments. We'll also need to set up the theme to include the CSS in releases.

Closes #73 .

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Run `npm run build` and make sure the CSS files are generated in the `/styles/` directory. Note: It's expected that there will be errors for the RTL files if there aren't corresponding CSS files -- so `npm run build` needs to be run twice, once to first generate each standard CSS file, and the second to generate the RTL files based on each of those CSS files. 
3.

### Other information:
